### PR TITLE
Setting Label widget's font properties now works in Gtk backend

### PR DESCRIPTION
**Pull request by ArchKudo**
## In this PR:
1. Implement `toga_gtk.widgets.Label.set_font`
2. Added tests
3. Organized imports

## PR Checklist:

- [x] All new features have been tested
- [ ] All new features have been documented
- [x] I have read the **CONTRIBUTING.md** file
- [x] I will abide by the code of conduct

### CODE REVIEW

**Comment by freakboy3742** [comment]
The code looks good, except for the test failure...

**Comment by ArchKudo** [comment]
@freakboy3742 tl;dr: For the tests to pass, xvfb-run is required. More at [pygobject docs](https://pygobject.readthedocs.io/en/latest/guide/testing.html)  
 Here's a gist of response I  got from the gtk+ irc:

> ArchKudo	Hi all! Calling the `get_pango_context` on a Gtk widget returns a NoneType in ci so I was wondering whether a headless display server was required for it to return a pango context object? I had similar results when using `get_style_context`. Background info: I am using pygobject, and the tests work fine when running on Gnome.	09:22
> lazka	ArchKudo, https://pygobject.readthedocs.io/en/latest/guide/testing.html	09:26
> ArchKudo	lazka: I did see the specific documentation page before, but I wanted to know whether there was a way of being certain that those two api require using xvfb?	09:34
> lazka	ArchKudo, without a display server the call to Gtk.init() in the Python overrides will silently fail. So any use of gtk api isn't valid	10:06
> lazka	for pango alone, I don't know, but if you use gtk api on the way you need it anyway	10:06
> lazka	ArchKudo, if you are on X11 you can test on your machine by unsetting DISPLAY, "DISPLAY= ./foo.py"	10:08
> ArchKudo	lazka: Thanks for replying!	10:13

[Or the unabridged version](https://quodlibet.duckdns.org/irc/foo/gtk%2B_2018-04-28.log.html)
Also I tried running `test_label.py` with `DISPLAY` unset under X and it gave me the following result:

```bash
~/Code/toga/src/gtk/tests/widgets gtk_set_font
toga ❯ DISPLAY= python test_label.py
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
```
as expected.

**Comment by freakboy3742** [comment]
Hrm; ok... that's annoying. In which case, we'll need an update to our [comb definition for GTK testing](https://github.com/pybee/comb/tree/pybee/tasks/gtk-py34).

**Comment by ArchKudo** [comment]
@freakboy3742 Thanks for replying, I can think of two ways to proceed:
1. Modifying the `run.sh` script i.e:
    - Add xvfb to dependencies
    - Replace command by `xvfb-run -a python setup.py test`
2. Or, use `pyvirtualdisplay`  a python wrapper for `xvfb` i.e:
    - Add `xvfb` and `pyvirtualdisplay` to dependencies
    - Use `pyvirtualdisplay.Display` in toga's Gtk+ tests

File: src/gtk/toga_gtk/widgets/label.py [hunk 1]
```diff
@@ -1,9 +1,7 @@
 from gi.repository import Gtk
 from travertino.size import at_least
-
-from toga.constants import *
-from toga_gtk.libs import gtk_alignment
 from toga_gtk.color import native_color
```
Comment (brutusthebee) [inline src/gtk/toga_gtk/widgets/label.py]: At column 1: (F401) 'toga_gtk.color.native_color' imported but unused

**Comment by ArchKudo** [comment]
From what I can tell, using Pango is better suited for dynamically setting the font of a widget as [Pango context](https://lazka.github.io/pgi-docs/#Gtk-3.0/classes/Widget.html#Gtk.Widget.get_pango_context) can listen to widget attribute changes also, the context is available throughout the widget's lifetime as opposed inherently static nature of CSS. Also fwiw, it seems to be the recommended way to change a font reading the source code of this Gtk [demo](https://gitlab.gnome.org/GNOME/gtk/blob/master/demos/gtk-demo/font_features.c) and, [lollypop](https://gitlab.gnome.org/World/lollypop/tree/master)(an actively maintained pygobject-based music player) which uses [css](https://gitlab.gnome.org/World/lollypop/blob/master/data/application.css) for theming but, Pango for setting [fonts](https://gitlab.gnome.org/search?utf8=%E2%9C%93&search=get_pango_context&group_id=&project_id=274&search_code=true&repository_ref=master)

**Comment by freakboy3742** [comment]
I'm not sure I follow why we'd need to listen to widget attribute changes. What would be changing in a font definition in response to a different widget attribute changing?

Related to #397 - I'd be interested to seeing a performance comparison here, too - are font changes via pango faster or slower than the equivalent changes via a CSS `font` declaration?

**Comment by freakboy3742** [comment]
Font setting on GTK has now been implemented in #1203.

**Pull request closed without merging**